src/server/services/validation-rules.js

```diff
@@ -1,105 +1,35 @@
-const XLSX = require('xlsx')
-const { merge } = require('lodash')
 
-const { bufferForUpload } = require('./persist-upload')
-const { DATA_SHEET_TYPES } = require('./records')
-const { templateForPeriod } = require('./get-template')
+const srcRules = require('../lib/templateRules.json')
+const srcDropdowns = require('../lib/templateDropdowns.json')
 
-const COLNAMES = makeColNames()
-const STATES = ['AL', 'AK', 'AS', 'AZ', 'AR', 'CA', 'CO', 'CT', 'DE', 'DC', 'FM', 'FL', 'GA', 'GU', 'HI', 'ID', 'IL', 'IN', 'IA', 'KS', 'KY', 'LA', 'ME', 'MH', 'MD', 'MA', 'MI', 'MN', 'MS', 'MO', 'MT', 'NE', 'NV', 'NH', 'NJ', 'NM', 'NY', 'NC', 'ND', 'MP', 'OH', 'OK', 'OR', 'PW', 'PA', 'PR', 'RI', 'SC', 'SD', 'TN', 'TX', 'UT', 'VT', 'VA', 'VI', 'WA', 'WV', 'WI', 'WY']
+function generateRules () {
+  const rules = JSON.parse(JSON.stringify(srcRules))
```

Comment:
What happens if you omit the JSON.stringify/parse calls here?  I think it should be possible to avoid this extra serialize/deserialize pass.